parts/kubernetesmasterresources.t

```diff
@@ -395,6 +396,7 @@
           {{end}}
         },
         "storageProfile": {
+          {{GetEtcdDisk}}
```

Comment:
Correct, functions are only needed where it is not possible to get information from the model, or the formatting is not easy to express in the templates (bz, managed disks vs, std disk formate, etc).